frontend/src/employee-mobile-frontend/components/attendances/actions/MarkAbsentBeforehand.tsx

```diff
@@ -209,3 +351,35 @@ const TimeInputWrapper = styled.span`
 const Separator = styled.span`
   padding-bottom: 5px;
 `
+
+function groupAbsencesByDateRange(absences: Success<Absence[]>): LocalDate[][] {
+  const sorted = orderBy(absences.value, ['date'])
+  const absencesGroupedByDate: Absence[][] = [[sorted[0]]]
+  let previousDate: LocalDate = absencesGroupedByDate[0][0].date
+  for (const absence of sorted) {
+    if (
+      differenceInDays(
```

Comment:
Lets not use date-fns here. LocalDate should have all the necessary methods. It may use date-fns internally.